### ISSUE TITLE
[BUG] Unit test fails at Travis.ci due to SQL deprecation warning

### ISSUE THREAD

**Issue by zobayer1** [zobayer1/elastic-migrate#97]
### TL;DR
Travis.ci unit test fails due to deprecation warning from SQLAlchemy's Table.exists() method

**Expected behavior**
All unit tests should pass

**Observed behavior**
Unit tests on DB manager is failing with the following error

    E       sqlalchemy.exc.SADeprecationWarning: The Table.exists() method is deprecated and will be removed in a future release.  Please refer to Inspector.has_table(). (deprecated since: 1.4)
    ../../../virtualenv/python3.6.7/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py:41: SADeprecationWarning

### Reproduction
1. Go to pull requests made by pyup bot.
2. Check Travis CI error logs.

**Additional information**
The error occurs due to supporting an upgraded version of SQLAlchemy.
